servers/src/main/java/tachyon/master/file/FileSystemMaster.java

```diff
@@ -248,12 +249,14 @@ public void stop() throws IOException {
    * @param fileId the file id
    * @param length the length of the file
    * @return true on success
-   * @throws SuspectedFileSizeException
+   * @throws InvalidFileSizeException
```

Comment:
it would be great if you could add missing exception descriptions for this method JavaDoc